src/API/Site/Controllers/MerchantCenter/ShippingRateSuggestionsController.php

```diff
@@ -0,0 +1,148 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\BaseController;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\ShippingRateSchemaTrait;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\TransportMethods;
+use Automattic\WooCommerce\GoogleListingsAndAds\Internal\Interfaces\ISO3166AwareInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+use Automattic\WooCommerce\GoogleListingsAndAds\Shipping\ShippingZone;
+use WP_REST_Request as Request;
+use WP_REST_Response as Response;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class ShippingRateSuggestionsController
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter
+ *
+ * @since x.x.x
+ */
+class ShippingRateSuggestionsController extends BaseController implements ISO3166AwareInterface {
```

Comment:
The ISO3166AwareInterface was used for converting a country code to country name, we can remove that from this class and from the ShippingRateController as well. Also the import line at the top of the file can be removed.

Reply:
Actually it seems like we also use it for validating the country code, so you can ignore this comment.